extension/content_script.js

```diff
@@ -314,27 +297,26 @@ function highlightKeyWords(container, start, end) {
 			start: word_start,
 			length: word_len
 		}], {
-			className: keywordStyle
+			className: "keywordClass"
 		});
 	}
+	$('.keywordClass').addClass(keywordStyle) 
 };
 
 /*
 Fade the current tracker indicator according to the calculated speed.
 */
 function fadeTracker() {
-	let displaySettings = display.getSettings();
-	let keywordStyle = "keyWord"+displaySettings[0];
 	fadeElement($("mark"));
-	fadeElement($("." + keywordStyle));
+	fadeElement($(".keywordClass"));
```

Comment:
- please turn "keywordClass" to a variable.
- and / or create a function to get the keyword / tracker els. If someday want to change the css class, then just swap out one place.